packages/p2p/src/components/orders/order-details/order-actions-block.jsx

```diff
@@ -149,17 +149,21 @@ const OrderActionsBlock = ({ cancelPopup, order_details, showPopup }) => {
             <React.Fragment>
```

Comment:
This is unrelated to your card, please undo these changes.

Reply:
> This is unrelated to your card, please undo these changes.

Done